### DISCUSSION TITLE
looking for part of the lps code

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1392] [Q&A]
Hi, Where can I find the lps code about determine the range with each anchor ? Is go to the lps node firmware or crazyflie firmware ? Thanks for help.

**Comment by knmcguire** [comment]
In the Loco deck code, one of the logging variables are the ranges of the Crazyflie to the anchors. You will need to go through the code from there though yourself.

https://github.com/bitcraze/crazyflie-firmware/blob/d8fd72fcb7ccff95d3f2e7a03d2f40c289f6d982/src/deck/drivers/src/lpsTwrTag.c#L625
